dc-model-jackson/src/test/resources/serializedTestObjects/webpage.json

```diff
@@ -1,67 +1,96 @@
-				"content": [{
-					"type": "text",
-					"text": ""
-				}]
-			}]
-		}
-	},
-	"identifiers": [],
-	"label": {
-		"de_DE": "Meine Homepage"
-	},
-	"type": "ENTITY_PART",
-	"entityPartType": "WEBPAGE",
-	"text": {
-		"de_DE": {
-			"type": "doc",
-			"content": [{
-				"type": "heading",
-				"content": [{
-					"type": "text",
-					"text": "Bayerische Staatsbibliothek",
-					"marks": [{
-						"type": "strong"
-					}]
-				}],
-				"attrs": {
-					"level": 4
-				}
-			}, {
-				"type": "paragraph",
-				"content": [{
-					"type": "text",
-					"text": "Internet:",
-					"marks": [{
-						"type": "strong"
-					}]
-				}, {
-					"type": "text",
-					"text": "   "
-				}, {
-					"type": "text",
-					"text": "www.bsb-muenchen.de",
-					"marks": [{
-						"type": "link",
-						"attrs": {
-							"href": "https://www.bsb-muenchen.de/"
-						}
-					}]
-				}]
-			}, {
-				"type": "iframe",
-				"attrs": {
-					"src": "https://external.content.org/index.php?language=de&action=test",
-					"width": "90%",
-					"height": "150px"
-				}
-			}]
-		}
-	}
-}
\ No newline at end of file
+  "description": {
+    "de_DE": {
+      "type": "doc",
+      "content": [
+        {
+          "type": "paragraph",
+          "content": [
+            {
+              "type": "text",
+              "text": ""
+            }
+          ]
+        }
+      ]
+    }
+  },
+  "identifiers": [],
+  "label": {
+    "de_DE": "Meine Homepage"
+  },
+  "type": "ENTITY_PART",
+  "entityPartType": "WEBPAGE",
+  "text": {
+    "de_DE": {
+      "type": "doc",
+      "content": [
+        {
+          "type": "heading",
+          "content": [
+            {
+              "type": "text",
+              "text": "Bayerische Staatsbibliothek",
+              "marks": [
+                {
+                  "type": "strong"
+                }
+              ]
+            }
+          ],
+          "attrs": {
+            "level": 4
+          }
+        },
+        {
+          "type": "paragraph",
+          "content": [
+            {
+              "type": "text",
+              "text": "Internet:",
+              "marks": [
+                {
+                  "type": "strong"
+                }
+              ]
+            },
+            {
+              "type": "text",
+              "text": "   "
+            },
+            {
+              "type": "text",
+              "text": "www.bsb-muenchen.de",
+              "marks": [
+                {
+                  "type": "link",
+                  "attrs": {
+                    "href": "https://www.bsb-muenchen.de/"
+                  }
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "type": "image",
+          "attrs": {
+            "resourceId": "135ec10b-ac65-4217-83fc-db5e9ff62cac",
+            "linkNewTab": true,
+            "width": "33%",
+            "alignment": "left",
+            "url": "https://www.bsb-munchen.de/logo.png"
```

Comment:
Gibt es einen Grund, warum `bsb-munchen` hier einmal falsch geschrieben wird, und in Zeile 68 richtig? Z.B. in `WebpageFactory.java` steht es auch so 🤔

Reply:
Danke, hab den Tippfehler behoben :+1: